Gemfile

```diff
@@ -1,10 +1,10 @@
 source 'https://rubygems.org'
 
-
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.6'
 # Use sqlite3 as the database for Active Record
-gem 'sqlite3'
+# gem 'sqlite3'
+gem 'pg', '~> 0.20.0'
```

Comment:
@ArtOfCode-  @schas002  I am wondering what is the best way to let the user choose the db ... do you mean just keep the database yml commented and instruction on how to implement it?